packages/2018-neighborhood-development/src/components/App/index.js

```diff
@@ -17,15 +17,15 @@ import DiveDeeperIntoNeighborhoodData from '../DiveDeeperIntoNeighborhoodData';
 
 const App = () => (
   <PageLayout>
+    <ClassSizeAndQuality />
```

Comment:
I know why you did this, but now you have to put it back 😉

Reply:
Bahaha I got tired of scrolling!